mkosi

```diff
@@ -1945,8 +1956,11 @@ def unlink_try_hard(path):
 
 def empty_directory(path):
 
-    for f in os.listdir(path):
-        unlink_try_hard(os.path.join(path, f))
+    try:
+        for f in os.listdir(path):
+            unlink_try_hard(os.path.join(path, f))
+    except:
```

Comment:
Swallowing all exceptions is a code smell. Why is it a good idea to presume everything worked when it didn't?